*** tests/A-10-05-M-ok.out	2018-11-28 19:02:07.000000000 +0000
--- tests/A-10-05-M-ok.outhyp	2018-12-07 16:32:54.877610800 +0000
***************
*** 15,21 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
--- 15,22 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Entregar projecto: Operação inválida: O projecto 'P1' não existe na disciplina 'D1'.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
***************
*** 26,35 ****
  3 - Portal Pessoal
  5 - Portal do Estudante
  0 - Sair
! Escolha uma opção: Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
  5 - Portal do Estudante
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 27,36 ----
  3 - Portal Pessoal
  5 - Portal do Estudante
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
  5 - Portal do Estudante
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
