*** tests/A-10-35-M-ok.out	2018-11-29 22:49:26.000000000 +0000
--- tests/A-10-35-M-ok.outhyp	2018-12-07 16:33:02.979650400 +0000
***************
*** 17,60 ****
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
! 1 - Criar projecto
! 2 - Fechar projecto
! 3 - Ver submissões de um projecto
! 4 - Ver alunos de disciplina leccionada
! 5 - Ver resultados de um inquérito
! 0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
! 1 - Criar projecto
! 2 - Fechar projecto
! 3 - Ver submissões de um projecto
! 4 - Ver alunos de disciplina leccionada
! 5 - Ver resultados de um inquérito
! 0 - Sair
! Escolha uma opção: Menu principal
! 1 - Abrir
! 2 - Guardar
! 3 - Portal Pessoal
! 4 - Portal do Docente
! 0 - Sair
! Escolha uma opção: Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
  4 - Portal do Docente
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 17,43 ----
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Nome da disciplina: Nome do projecto: Nome da disciplina: Nome do projecto: Nome da disciplina: Nome do projecto: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Nome da disciplina: Nome do projecto: Nome da disciplina: Nome do projecto: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
! Fechar projecto: Operação inválida: A disciplina '0' não existe ou não é válida para o utilizador em causa.
! Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
! Menu principal
  1 - Abrir
  2 - Guardar
  3 - Portal Pessoal
  4 - Portal do Docente
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
