*** tests/A-10-31-M-ok.out	2018-11-29 11:35:14.000000000 +0000
--- tests/A-10-31-M-ok.outhyp	2018-12-07 16:33:01.912628900 +0000
***************
*** 13,20 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Criar inquérito: Operação inválida: A disciplina 'D33' não existe ou não é válida para o utilizador em causa.
! Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
--- 13,19 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
***************
*** 22,29 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Criar inquérito: Operação inválida: O projecto 'P4' não existe na disciplina 'D2'.
! Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
--- 21,27 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
